### DISCUSSION TITLE
UGS Platform - Soft Reset Button

### DISCUSSION THREAD

**Discussion by art-and-piano** [winder/Universal-G-Code-Sender#1894] [Help] [answered]
Hello, could somesone tell me which commands the "Soft Rest" button in the UGS-platform is sending? And could it be possible to implement these commands as part of a macro? Thank you, Thomas

**Comment by breiler** [comment, marked as answer]
I will assume you mean the command for a GRBL-based controller.

We use the real time command CTRL-X (0x18). No it is not possible to use these as a part of a macro. These are real time commands which will be executed as soon as they are received by the controller and will not be a part of the controller command stream.

**Reply by art-and-piano** [reply]
Thank you!
